airbyte-config/config-models/src/main/java/io/airbyte/config/Configs.java

```diff
@@ -641,4 +719,10 @@ enum SecretPersistenceType {
     VAULT
   }
 
+  enum WorkerPlane {
+    CONTROL_PLANE,
+    DATA_PLANE,
+    COMBINED
```

Comment:
@gosusnp discussion from parent PR:
![image](https://user-images.githubusercontent.com/7798112/186284984-962f506e-2ee3-4e1c-b321-2fa3f9c29c24.png)

Reply:
Yeah I think you're right, `COMBINED` effectively means the same thing as `CONTROL_PLANE` the way I'm using it, so I can remove it